ops/reth-cross-test/verification/stress_runner.py

```
@@ -0,0 +1,121 @@
+                "gas": 21000,
+                "maxFeePerGas": (w3.eth.get_block("latest").get("baseFeePerGas", 1000000)) * 2, "maxPriorityFeePerGas": w3.eth.get_block("latest").get("baseFeePerGas", 1000000), "type": 2,
+                "nonce": nonce,
+                "chainId": w3.eth.chain_id,
+            }
+            signed = funded.sign_transaction(tx)
+            w3.eth.send_raw_transaction(signed.raw_transaction)
+            nonce += 1
+            time.sleep(0.05)
+        time.sleep(10)  # wait for funding txs to be included
+
+    def run_stage(self, concurrency: int, duration: int, contract_addr: str = None):
```

Ruff (0.15.7)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)